src/cryptography/fernet.py

```diff
@@ -157,8 +157,8 @@ def _decrypt_data(
 
 
 class MultiFernet(object):
-    def __init__(self, fernets_it: typing.Iterator[Fernet]):
-        fernets = list(fernets_it)
+    def __init__(self, fernets: typing.Iterable[Fernet]):
```

Comment:
Changing this back to match the docs?

Reply:
`list` isn't iterator, but it is iterable